<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDemo</class>
 <widget class="QMainWindow" name="RoomDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoomDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_main" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1080</width>
      <height>670</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1080</width>
      <height>670</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1080</width>
      <height>670</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget_main{
  border: none;
  background-color: #ffffff;
}   </string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget_top" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">#widget_top{
	background-color: #006eff;
}</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_logo">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_title">
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Microsoft YaHei&quot;;
color: #ffffff;</string>
          </property>
          <property name="text">
           <string>title</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_min">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;Microsoft YaHei&quot;;
	border: none;
	image: url(:/RoomService/min-button.png);
	background-position: center;
}
QPushButton:hover{
	image: url(:/RoomService/min-hover.png);
}
QPushButton:pressed{
	image: url(:/RoomService/min-hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_max">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;Microsoft YaHei&quot;;
	border: none;
	image: url(:/RoomService/max-button.png);
	background-position: center;
}
QPushButton:hover{
	image: url(:/RoomService/max-hover.png);
}
QPushButton:pressed{
	image: url(:/RoomService/max-hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_close">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	font: 10pt &quot;Microsoft YaHei&quot;;
	border: none;
	image: url(:/RoomService/close-button.png);
	background-position: center;
}
QPushButton:hover{
	image: url(:/RoomService/close-hover.png);
}
QPushButton:pressed{
	image: url(:/RoomService/close-hover.png);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>horizontalSpacer_6</zorder>
       <zorder>label_logo</zorder>
       <zorder>label_title</zorder>
       <zorder>btn_close</zorder>
       <zorder>btn_min</zorder>
       <zorder>btn_max</zorder>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_bottom" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_member" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>630</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;
</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>3</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_main_share" native="true">
          <property name="styleSheet">
           <string notr="true">#widget_main_share
{	background-color: rgb(247,248,250);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_message" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>630</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #ffffff;</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resources/RoomService/RoomService.qrc"/>
 </resources>
 <connections/>
</ui>
